skaffold.yaml

```diff
@@ -1185,52 +1185,14 @@ profiles:
   activation:
   - env: HA_MODE=single-zone
   patches:
-  - op: replace
-    path: /deploy/helm/releases/0/setValueTemplates
-    value:
-      global:
-        gardenlet:
-          imageVectorOverwrite: |
-            images:
-            - name: gardenlet
-              repository: localhost:5001/eu_gcr_io_gardener-project_gardener_gardenlet
-              tag: "{{.IMAGE_TAG6}}"
-            - name: gardener-resource-manager
-              repository: localhost:5001/eu_gcr_io_gardener-project_gardener_resource-manager
-              tag: "{{.IMAGE_TAG7}}"
-            - name: gardener-seed-admission-controller
-              repository: localhost:5001/eu_gcr_io_gardener-project_gardener_seed-admission-controller
-              tag: "{{.IMAGE_TAG8}}"
-          config:
-            seedConfig:
-              spec:
-                highAvailability:
-                  failureTolerance:
-                    type: node
+  - op: add
```

Comment:
No 😓  the `path` must be something from skaffold's `Config` kind.
But we could add `setValues` directives instead of replacing the `setValueTemplates` 🥳